dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessServiceImpl.java

```diff
@@ -276,6 +275,9 @@ public class ProcessServiceImpl implements ProcessService {
     @Autowired
     private CuringParamsService curingGlobalParamsService;
 
+    @Autowired
+    private ProcessService processService;
```

Comment:
Quote itself?

Reply:
This used to use transaction in spring, otherwise we need to use SprintContext.getBean like before.